README.md

```
@@ -1,65 +1,65 @@
-[pr]: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
-[git]: https://git-scm.com
-[node]: https://nodejs.org/
-[ts]: https://www.typescriptlang.org/
-[discord.js]: https://discord.js.org/
-[eslint]: https://eslint.org/
-[prettier]: https://prettier.io/
-[ai-sdk]: https://ai-sdk.dev/
-[bun]: https://bun.sh/
-[mem0]: https://mem0.ai/
-[exa]: https://exa.ai/
+<h1 align="center">AI Discord Bot</h4>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: h1

(MD033, no-inline-html)